lib/Conversion/TorchToLinalg/DataMovement.cpp

```diff
@@ -33,6 +33,8 @@ using namespace mlir;
 using namespace mlir::torch;
 using namespace mlir::torch::Torch;
 
+#include <iostream>
```

Comment:
Code in torch-mlir should never import `iostream`

https://llvm.org/docs/CodingStandards.html#include-iostream-is-forbidden